packages/fluentui/docs/.storybook/main.js

```diff
@@ -1,40 +1,51 @@
+// @ts-check
+
 const path = require('path');
-const { TsconfigPathsPlugin } = require('tsconfig-paths-webpack-plugin');
+
+const { registerTsPaths, registerRules, rules } = require('@fluentui/scripts-storybook');
```

Comment:
_nit:_ Order alphabetically

```suggestion
const { registerRules, registerTsPaths, rules } = require('@fluentui/scripts-storybook');
```